### ISSUE TITLE
Introduce capability for using the IDE

### ISSUE THREAD

**Issue by jasonbahl** [wp-graphql/wpgraphql-ide#161]
Currently the IDE is limited to users with the `manage_options` capability. 

It would be nice to have a specific capability dedicated to interacting with the IDE that any user role can be allowed to interact with the IDE.

**Comment by josephfusco** [comment]
[This release](https://github.com/wp-graphql/wpgraphql-ide/pull/187) fixes an issue with the capability in the IDE. Once that is merged this can be closed.

**Comment by josephfusco** [comment]
This has been included in https://github.com/wp-graphql/wpgraphql-ide/releases/tag/v2.1.5
